doc_src/index.hdr.in

```diff
@@ -17,7 +17,199 @@
 
 \section introduction Introduction
 
-This is the documentation for `fish`, the friendly interactive shell. `fish` is a user friendly commandline shell intended mostly for interactive use. A shell is a program used to execute other programs. For the latest information on `fish`, please visit the <a href="https://fishshell.com/">`fish` homepage</a>.
+This is the documentation for *fish* the <b>f</b> riendly <b>i</b> nteractive <b>sh</b> ell.
+
+A shell is a commandline interpreter. It reads text input from the commandline and interpretes it as commands to operating system, see: <a href="#shell">What is a Shell</a>.
+
+So shells serve as user-interface between applications and the operating system. There are many different shells. They differ on how this interface is implemented. *fish* specializes in the following ways:
+
+- <b>Extensive UI</b>: *fish* supports the user with syntax highlighting, autosuggestions, tab completions and selections lists, that can be navigated and filtered, see: <a href="#autosuggestions">Autosuggestions</a> and <a href="#completion">Tab Completions</a>.
+
+- <b>No further configuration is needed</b>: *fish* comes preconfigured so that it will be an efficient helper on the commandline out of the box.
+
+- <b>Add new commands easily</b>: in *fish* new commands can be added on the fly. The syntax is easy to learn and there is no administrative overhead, see: <a href="#syntax-function">Functions</a>.
+
+- <b>Interactive shell</b>: *fish* focuses on commands that will be run in an interactive session: While it supports job control for external commands, its own commands share the process of the shell, that started them.
+
+<a href="#introduction"><-</a><hr>
+
+
+\section shell What is a Shell
```

Comment:
I think this section needs reworking. I would drop the "shell standards" paragraph, as it raises more questions than it answers; likewise the "manual pages" paragraph is much more about UNIX than it is about the shell.

Reply:
Okay here comes my rewrite of this whole section:

"## What is a Shell

This section is about some basic features of fish. These features are related to the fact that fish is a command-line shell.
 
- <a href="#purpose-of-a-shell">Purpose of a Shell</a>: explains the main task of a shell: being the middleman between users and applications
- <a href="#shell-scripting">Shell Scripting</a>: shells also serve as a programing language

### The Purpose of a Shell

A shell can be seen as the outermost layer to the operating system of a computer: 

When a new programm is installed on a computer, it delivers an executeable file, that is responsible to setup and run the application. But this file does not know how to communicate with the operating system. 

This exactly is the task of a shell such as fish: to serve as the middle man between any application, the user and the operating system.

- The application comes with a command (the name of an executeable file)

- The user can enter that command into the shell

- When the shell gets the command it searches for an executeable file of that name

- When the executeable file is found, it gets handed over to the operating system and starts a process in the operating system

- During the runtime of the process the executable may send data as a continous stream. That data is then picked up by the shell and released to an output channel. That output channel is normally the interactive shell session, but can also be redirected to a file.

- When the process ends in the operating system, it exits with a numeric exit code. This exit code gets handed over to the shell and the shell may react to it by writing an error message to the user.

### Shell Scripting

Besides dealing with other applications a command-line shell such as fish also provides a language itself, that enables the user to communicate directly with the operating system.

Shell scripts are often used to bundle and automate sequences of tasks under one command. 

So fish is also a scripting language and scripting in fish can be done in an interactive way.

fish can also execute scripts written in other programming languages, see the section on shell scripting further down."

Now this is just a first draft to replace the section. Does that sound more like what you were thinking of? I just want to give an idea of what a shell is all about in that section.

Reply:
Yes, that's more the general idea.